test/summary/clearlyDefined.js

```diff
@@ -266,11 +282,11 @@ describe('ClearlyDefined Gem summarizer', () => {
   })
 })
 
-function setupGem(releaseDate, license, sourceInfo) {
+function setupGem(releaseDate, licenses, sourceInfo) {
```

Comment:
propose just have this take a string or an array for the second arg. If it is a string then set the `license` prop. If an array, set the `licenses` prop

Reply:
done